modules/extensions-support/src/main/java/org/mule/module/extension/internal/util/IntrospectionUtils.java

```diff
@@ -42,7 +42,7 @@
  *
  * @since 3.7.0
  */
-public class IntrospectionUtils
+public abstract class IntrospectionUtils
```

Comment:
This should not be abstract. Having an abstract class means that the class is designed to be extended in order to use it. Add a private constructor.